### DISCUSSION TITLE
Ability to regenerate a post slug from new title

### DISCUSSION THREAD

**Discussion by adamkdean** [forem/forem#15329] [Feature Requests]
(Feature request)

Quite simply, when you create a post, the slug is set to the title plus four characters. But if you update the title, the old slug remains. It'd be great to be able to regenerate this.

It'd be even better if the old one remained, and then did a 301 or 302 to the new slug, but it'd be fine to say "hey, if you change this, old links will break with a [YES I KNOW] button etc".

Just an idea.

**Comment by axsharma** [comment]
Any updates on this? I'd love to see this feature on dev.to. Especially when editing an unpublished draft - it is only natural the title will change several times before the post goes live (therefore a 301/302 is irrelevant).

**Comment by LucasKuhn** [comment]
I have the same issue as @axsharma, I usually make a quick draft and want to change the automatically generated slug on Dev.to

**Comment by Link2Twenty** [comment]
Hey, a good way to get an issue like this to be noticed is to make a post over on <https://forem.dev> explaining the desired feature and asking for people's thoughts.
